queue/driver.go

```diff
@@ -25,6 +25,9 @@ type remoteQueueDriver interface {
 	JobStats(context.Context) <-chan amboy.JobStatusInfo
 	Complete(context.Context, amboy.Job) error
 
+	// kim: TODO: maybe remove
```

Comment:
the queue drivers are all internal, and at some point we'll restructure the mongod queues to avoid the remote queue+driver vestige... In the mean time, I think it's fine.